Update the README file and migrate OAuth2 Mock to this repository (#1)
kyma-project/networking-dev-tools: oauth2-mock/README.md

# OAuth2 Mock

## Description

> This is a simple OAuth2 mock server that can be used for testing purposes. It allows you to simulate an OAuth2 server that can issue tokens and validate them.

Reviewer: ```suggestion
This is a simple OAuth2 mock server that you can use for testing purposes. It allows you to simulate an OAuth2 server that can issue and validate tokens.
```
